recipes/libmatio/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+./configure --enable-mat73 --enable-extended-sparse --prefix="${PREFIX}" --with-zlib="${PREFIX}" --with-hdf5="${PREFIX}"
+make
+
+if [[ `uname` == 'Darwin' ]];
+then
+	eval DYLD_FALLBACK_LIBRARY_PATH=$PREFIX/lib make check
+else
+	make check
```

Comment:
Again. Using `gcc` on Linux is ok. Just make sure `libgcc` is a runtime dependency. Also, make sure it is Linux only.